app/src/main/java/org/simple/clinic/settings/SettingsRepository.kt

```diff
@@ -0,0 +1,10 @@
+package org.simple.clinic.settings
+
+import io.reactivex.Single
+
+class SettingsRepository {
```

Comment:
Can we not make this an interface, the implementation details can come later when implementing the UI?

Reply:
Done.